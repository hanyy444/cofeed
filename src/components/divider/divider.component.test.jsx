import { render, screen } from "@testing-library/react";
import Divider from "./divider.component";

describe('Test divider component', () => { 
    it('Should render without crashing', () => {
        expect.assertions(2)
        render(<Divider/>)
        const divider = screen.getByTestId('divider')
        expect(divider).toBeInTheDocument()
        expect(divider).toMatchSnapshot()
    })
 })