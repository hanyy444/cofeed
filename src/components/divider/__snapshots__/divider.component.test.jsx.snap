// Vitest Snapshot v1

exports[`Test divider component > Should render without crashing 1`] = `
<hr
  class="solid"
  data-testid="divider"
/>
`;
